src/lib/interfaces/digital-asset-deployment.ts

```diff
@@ -0,0 +1,27 @@
+import { LSP7 } from '../../tmp/LSP7';
+import { LSP8 } from '../../tmp/LSP8';
+
+export enum ContractNames {
+  LSP7_DIGITAL_ASSET = 'LSP7DigitalAsset',
+  LSP8_DIGITAL_ASSET = 'LSP8IdentifiableDigitalAsset',
+}
+
+export interface DigitalAssetDeploymentOptions {
+  ownerAddress: string;
+  name: string;
+  symbol: string;
+}
+
+export interface LSP7DigitalAssetDeploymentOptions extends DigitalAssetDeploymentOptions {
+  isNFT: boolean;
+}
+
+export enum DeploymentStatus {
+  PENDING = 'PENDING',
+  COMPLETE = 'COMPLETE',
+}
```

Comment:
This is duplicate with `src/lib/interfaces/deployment-events.ts:L12`

Can you use the same interface instead?

Reply:
Ah cool yes i missed this, thanks :)

Reply:
Ah cool yes i missed this, thanks :)